http-kernel-fixtures/sub-folder/cookie_page4.php

```diff
@@ -0,0 +1,21 @@
+<?php
+    $resp = new Symfony\Component\HttpFoundation\Response();
+    if (method_exists('Symfony\Component\HttpFoundation\Cookie', 'create')) {
+        $cook = Symfony\Component\HttpFoundation\Cookie::create('srvr_cookie', 'srv_var_is_set', 0, '/sub-folder/');
+    } else {
+        $cook = new Symfony\Component\HttpFoundation\Cookie('srvr_cookie', 'srv_var_is_set', 0, '/sub-folder/');
+    }
+    $resp->headers->setCookie($cook);
+?>
+<!doctype html public "-//w3c//dtd xhtml 1.1//en" "http://www.w3.org/tr/xhtml11/dtd/xhtml11.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml" xml:lang="ru">
+<head>
+    <title>Basic page with cookie set in sub-directory from server side</title>
+    <meta http-equiv="content-type" content="text/html;charset=utf-8"/>
+    <script>
+    </script>
```

Comment:
the HTML returned here must be the same than in the corresponding `web-fixtures` file.